.gitignore

```
@@ -1,2 +1,3 @@
 .idea
 node_modules
+.cache
```

Saltó el warning de la última línea vacía

```suggestion
.cache

```